examples/getting-started-with-serverless-framework/README.md

```diff
@@ -0,0 +1,143 @@
+1. [Set up local project](#2-set-up-local-project)
+1. [Create a Slack app](#3-create-a-slack-app)
+1. [Deploy to AWS Lambda](#4-deploy-to-aws-lambda)
+1. [Update Slack app settings](#5-update-slack-app-settings)
+1. [Test your Slack app](#6-test-your-slack-app)
+1. [Run on local machine](#7-run-on-local-machine)
+
+## 1. Set up AWS credentials
+
+### Install AWS CLI
+
+Follow the guide to [install the AWS CLI v2 for macOS, Windows, or Linux][aws-cli-install].
+
+### Configure AWS CLI
+
+Follow the guide to [create a new IAM User][aws-cli-configure-user] then configure your local machine with the command:
+
+```zsh
+aws configure
+```
+
+## 2. Set up local project
+
+You can install the app's local development dependencies with the following command:
+
+```zsh
+npm install
+```
+
+You may also want to [install ngrok][ngrok-install] to start a local tunnel for local development.
+
+## 3. Create a Slack app
+
+### Create an app on api.slack.com
+
+1. Go to https://api.slack.com/apps
+1. Select **Create New App**
+    * Name your app, _don't worry you can change it later!_
+1. Select **OAuth & Permissions**
+    1. Scroll down to **Bot Token Scopes**
+    1. Add the following bot scopes:
+        1. Add the scope `app_mentions:read`
+        1. Add the scope `channels:history`
+        1. Add the scope `chat:write`
+        1. Add the scope `groups:history`
+        1. Add the scope `im:history`
+        1. Add the scope `mpim:history`
+    1. Select **Install App to Workspace** at the top of the page
+
+### Export environment variables
+
+```zsh
+export SLACK_SIGNING_SECRET=<your-signing-secret> # Slack app settings > "Basic Information"
+export SLACK_BOT_TOKEN=<your-xoxb-bot-token>      # Slack app settings > "OAuth & Permissions"
+```
+
+## 4. Deploy to AWS Lambda
+
+Run the following the command to deploy to AWS Lambda:
+
+```zsh
+npx serverless deploy
+# ...
+# endpoints:
+#  POST - https://d5c0t1xad4.execute-api.us-east-1.amazonaws.com/dev/slack/events
+```
+
+_Please note the endpoint `https://{your-domain}.amazonaws.com/dev/slack/events` because we'll use it in the next section._
+
+## 5. Update Slack app settings
+
+Now that your Slack app is deployed, you can register your AWS Lambda endpoint with the Slack API:
+
+1. Go to https://api.slack.com/apps
+1. Select your app
+1. Select **Event Subscriptions**
+    1. Enable **Events**
+    1. Set the **Request URL** to `https://{your-domain}.amazonaws.com/dev/slack/events`
+    1. Scroll down to **Subscribe to Bot Events**
+    1. Add the following bot events:
+        - `app_mention`
+        - `message.channels`
+        - `message.groups`
+        - `message.im`
+        - `message.mpim`
+    1. Select **Save Changes**
+
+## 6. Test your Slack app
+
+You can test your app by opening a Slack workspace and saying "hello" (lower-case):
+
+> 💬 hello
+>
+> 🤖 Hey there @Jane!
+
+_Remember, your app must be in the channel or DM where you say hello._
+
+## 7. Run on local machine
```

Comment:
This section feels out of place (it feels like developing locally should come _before_ deploying it to Lambda). I'd suggest not making it a "step" of the tutorial unless I'm missing something. You could include it as a section without the number or just link out to the relevant part of the slack.dev tutorial.

Reply:
Maybe it is because the guide and sample app are all about lambda deploy that local machine running feels like an afterthought. Personally, I think it is fine at the end, but maybe add a blurb about why you might want to run it locally. Or remove it 🤷

Reply:
I agree, it's a little out-of-place. This section should come after creating a Slack app, but that complicates configuring the app for deployment (the main purpose of this repo).

I think it's important to mention how to run the app locally. Running a serverless app isn't the same as a traditional app, so some direction would save developers time.

@shaydewael what are your thoughts on renaming the section to "Develop" and leaving it at the end? Developing would be the next/last step after deploying:
> **7. Develop on local machine**

Reply:
I think it flows much better with the structure you have now. I don't know why I originally said it could be removed (that would feel weird). LGTM